source/_patterns/00-atoms/global/colors.mustache

```diff
@@ -1,8 +1,67 @@
 <ul class="sg-colors">
 
+	<h2>Primary Theme Colors</h2>
+
+	<li>
+		<span class="sg-swatch bg--emperor" title="" style="background: "></span>
+		<span class="sg-label small">Emperor</span>
+		<span class="sg-label small"> #4b207f</span>
+	</li>
+
+	<li>
+		<span class="sg-swatch bg--earth" title="" style="background: "></span>
+		<span class="sg-label small">Earth</span>
+		<span class="sg-label small"> #5e3929</span>
+	</li>
+
+	<li>
+		<span class="sg-swatch bg--grapevine" title="" style="background: "></span>
+		<span class="sg-label small">Grapevine</span>
+		<span class="sg-label small"> #7f264a</span>
+	</li>
+
+	<li>
+		<span class="sg-swatch bg--denim" title="" style="background: "></span>
+		<span class="sg-label small">Denim</span>
+		<span class="sg-label small">#2f557f</span>
+	</li>
+
+	<li>
+		<span class="sg-swatch bg--campfire" title="" style="background: "></span>
+		<span class="sg-label small">Campfire</span>
+		<span class="sg-label small">#e36520</span>
+	</li>
+
+	<li>
+		<span class="sg-swatch bg--treefrog" title="" style="background: "></span>
+		<span class="sg-label small">Treefrog</span>
+		<span class="sg-label small">#448d21</span>
+	</li>
+
+	<li>
+		<span class="sg-swatch bg--ming" title="" style="background: "></span>
+		<span class="sg-label small">Ming</span>
+		<span class="sg-label small">#3e8391</span>
+	</li>
+
+
+</ul>
+
+
+<ul class="sg-colors">
```

Comment:
@SRHannah18 To add a little spacing to these and the above sections, add [one of these spacing classes](https://github.com/adventistchurch/alps/blob/master/source/css/_modifier.spacing.scss#L126) to this parent element.